src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/onestatementperline/InputOneStatementPerLineTryWithResources.java

```diff
@@ -0,0 +1,33 @@
+//non-compiled with javac: Compilable with Java9
+package com.puppycrawl.tools.checkstyle.checks.coding.onestatementperline;
+
+import java.io.IOException;
+import java.io.PipedOutputStream;
+import java.io.OutputStream;
+
+public class InputOneStatementPerLineTryWithResources {
+
+    void method() throws IOException {
+        OutputStream s1 = new PipedOutputStream();
+        OutputStream s2 = new PipedOutputStream();
+        try (s1; s2; OutputStream s3 = new PipedOutputStream();) {
+        }
+        try (s1; OutputStream s4 = new PipedOutputStream(); s2;) {
+        }
+        try (s1; s2; OutputStream s5 = new PipedOutputStream()) {
+        }
+        try (s1; OutputStream s6 = new PipedOutputStream(); s2) {
+        }
+        try (
+           OutputStream s7 = new PipedOutputStream();OutputStream s8 = new PipedOutputStream();
```

Comment:
Please add a comment that there is a violation on this line.

Reply:
done